interfaces/telegram.py

```diff
@@ -23,6 +29,11 @@ def _init_handlers(self):
                 self._handle_message
             ),
 
+            MessageHandler(
```

Comment:
https://python-telegram-bot.readthedocs.io/en/stable/telegram.ext.commandhandler.html - there is a CommandHandler for commands

Reply:
fxd